_topic_maps/_topic_map_rosa.yml

```diff
@@ -221,65 +254,50 @@ Topics:
   Topics:
     - Name: Configuring custom domains for applications
       File: osd-config-custom-domains-applications
+# - Name: Application GitOps workflows
+#   File: rosa-app-gitops-workflows
+# - Name: Application logging
+#   File: rosa-app-logging
+# - Name: Applications
+#   File: rosa-apps
+# - Name: Application metrics and alerts
+#   File: rosa-app-metrics and alerts
+# - Name: Projects
+#   File: rosa-projects
+# - Name: Using the internal registry
+#   File: rosa-using-internal-registry
 ---
-Name: Logging
-Dir: logging
-Distros: openshift-rosa
-Topics:
-- Name: Installing the cluster logging service
-  File: rosa-install-logging
-- Name: Viewing cluster logs
-  File: rosa-viewing-logs
----
-Name: rosa CLI
-Dir: rosa_cli
-Distros: openshift-rosa
-Topics:
-- Name: Getting started with the rosa CLI
-  File: rosa-get-started-cli
-- Name: Managing objects with the rosa CLI
-  File: rosa-manage-objects-cli
-- Name: Checking account and version information with the rosa CLI
-  File: rosa-checking-acct-version-cli
-- Name: Checking logs with the rosa cli
-  File: rosa-checking-logs-cli
----
-Name: Configuring private connections
-Dir: cloud_infrastructure_access
+Name: Storage and registry
+Dir: storage
 Distros: openshift-rosa
 Topics:
-- Name: Configuring private connections
-  File: rosa-configuring-private-connections
-- Name: Configuring AWS VPC peering
-  File: dedicated-aws-peering
-- Name: Configuring AWS VPN
-  File: dedicated-aws-vpn
-- Name: Configuring AWS Direct Connect
-  File: dedicated-aws-dc
-- Name: Configuring a private cluster
-  File: rosa-private-cluster
+# - Name: Configuring the internal registry
+#   File: configuring-registry-operator
+- Name: Persistent storage
+  Dir: persistent_storage
+  Distros: openshift-rosa
+  Topics:
+  - Name: Persistent storage using AWS Elastic Block Store (EBS)
+    File: rosa-persistent-storage-aws-ebs
+  - Name: Persistent storage using AWS EFS
+    File: osd-persistent-storage-aws
```

Comment:
Some of the files are reused in both OSD and ROSA so the naming here is okay. We may eventually want to take a closer look a file naming for distros because it's only going to get more convoluted as we share more content.